e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/premium-payments-error-states/create-fee-record-correction-errors.spec.js

```diff
@@ -0,0 +1,90 @@
+import {
+  CURRENCY,
+  FEE_RECORD_STATUS,
+  FeeRecordEntityMockBuilder,
+  PaymentEntityMockBuilder,
+  PENDING_RECONCILIATION,
+  UtilisationReportEntityMockBuilder,
+} from '@ukef/dtfs2-common';
+import pages from '../../../../../pages';
+import USERS from '../../../../../../fixtures/users';
+import { NODE_TASKS } from '../../../../../../../../e2e-fixtures';
+import relative from '../../../../../relativeURL';
+import { errorSummary } from '../../../../../partials';
+import { getMatchingTfmFacilitiesForFeeRecords } from '../../../../../../support/utils/getMatchingTfmFacilitiesForFeeRecords';
+
+context('When fee record correction feature flag is enabled', () => {
+  context('PDC_READ users cannot create record correction requests with invalid fee selections', () => {
```

Comment:
Should this not read `PDC_RECONCILE users...` ? - PDC_READ users shouldn't be able to make a record correction request at all

```suggestion
  context('PDC_RECONCILE users cannot create record correction requests with invalid fee selections', () => {
```